variables.tf

```diff
@@ -39,18 +39,24 @@ variable "additional_ses_sendemail_account_ids" {
   default     = []
 }
 
-variable "aws_region" {
+variable "api_gateway_east_1_zone_id" {
   type        = string
-  description = "The AWS region to communicate with."
-  default     = "us-east-1"
+  description = "The Route 53 hosted zone ID for us-east-1 endpoints."
+  default     = "Z1UJRXOUMOOFQ8"
```

Comment:
Since we already specify the region for this deployment in the `aws_region` variable, this variable should be renamed `api_gateway_zone_id`.